celery/events/dispatcher.py

```diff
@@ -218,7 +218,10 @@ def extend_buffer(self, other):
 
     def close(self):
         """Close the event dispatcher."""
-        self.mutex.locked() and self.mutex.release()
+        try:
+            self.mutex.release()
+        except RuntimeError:
+            pass
```

Comment:
This is too broad. We need to catch specific exceptions.

Reply:
This just catch the `RuntimeError` from `RLock.release()`.
But I think maybe CI exception cause by RLock didn't release.